app/services/user_update.rb

```diff
@@ -140,6 +140,8 @@ def record_email_change_and_notify
 
     EventLog.record_email_change(user, email_change.first, email_change.last, current_user)
 
+    return if user.api_user?
```

Comment:
I know you're exercising this in controller tests but did you consider/is it worth unit testing it in `UserUpdateTest` too?

Reply:
That's a reasonable request - I think I'd got lulled into not doing it because there aren't many such unit tests at the moment!

Reply:
Addressed in https://github.com/alphagov/signon/pull/2593/commits/77b6ce3c63a8068b131aa1bb2060025c512970fa